helpers/dbload.ts

```diff
@@ -96,6 +99,34 @@ UserLesson.belongsTo(User, {
 
 sequelize.sync({ alter: !!process.env.ALTER_DB })
 
+/* istanbul ignore next */
+if (process.env.NODE_ENV === 'development') {
+  //add data after fresh install
+  Lesson.findAll({
+    limit: 1
+  }).then((lessons: Array<LesssonTypescriptTypes>) => {
```

Comment:
This codebase uses `async/await` rather than `.then` to handle promises, can you put your promises into an async await function?